src/index.js

```diff
@@ -1,3 +1,3 @@
 export * from "./index-fn.js";
 import Treecle from "./Treecle.js";
-export default Treecle;
\ No newline at end of file
+export default Treecle;
```

Comment:
Same for this. You don't need to send a PR for adding a line break at the end of a fole!